engine/src/main/java/io/camunda/zeebe/engine/state/instance/DbUserTaskState.java

```diff
@@ -31,12 +33,23 @@ public class DbUserTaskState implements UserTaskState, MutableUserTaskState {
 
   private final ColumnFamily<DbLong, UserTaskRecordValue> userTasksColumnFamily;
 
+  // key => job state
+  private final DbForeignKey<DbLong> fkUserTask;
+  private final UserTaskStateValue userTaskState = new UserTaskStateValue();
```

Comment:
:thinking: Would it be clearer if we named this variable `userTaskLifecycleState`? Maybe we can rename the class that way?

I'm suggesting this since very often when working with Jobs, I see a `jobState` variable, and I'm not sure if it's the `DbJobState` or the `JobStateValue` so I need to search the code to clarify.

Reply:
Oh yeah, I absolutely agree. I wasn't sure if I should stick to the naming we use in jobs so it's easier to make the connection. But if you dislike it there, I'll adjust this here for good 👍

Reply:
Done with https://github.com/camunda/zeebe/pull/15433/commits/0399aaf3a1e0b758d698835591dbe84170639b9d